test/e2e/utils/utils.go

```diff
@@ -51,9 +50,6 @@ func WaitForNodesWithResource(c clientset.Interface, res v1.ResourceName, timeou
 			for t := time.Now(); time.Since(t) < nodeListTimeout; time.Sleep(poll) {
 				nodelist, err := c.CoreV1().Nodes().List(context.TODO(), metav1.ListOptions{})
 				if err != nil {
-					if testutils.IsRetryableAPIError(err) {
-						continue
-					}
```

Comment:
I don't remember how we ended up with this function. Is it a copypaste from k8s/test/e2e?

Reply:
I think so yes. `IsRetryableAPIError()` was removed and the commit changed places like this too.

Reply:
https://github.com/kubernetes/kubernetes/commit/64c099d67069f85f2b9b78d7d12fd1a92fcc3d75